surf/src/diff.rs

```diff
@@ -124,6 +124,8 @@ pub enum FileDiff {
 pub struct Hunk {
     pub header: Line,
     pub lines: Vec<LineDiff>,
+    pub old_missing_eof: bool,
+    pub new_missing_eof: bool,
```

Comment:
Shouldn't this be on the file? There can only be one missing eof per file.

I'm also not convinced of the representation, as it doesn't represent a diff. I think something like

```rust
enum EofDiff {
  Addition, Deletion
}

enum FileDiff {
   Plain {
     ...
     eof: Option<EofDiff>
   }
}
```

Reply:
I added it in `ModifiedFile` struct instead as it's the only place we're using it currently, and also simpler.

Reply:
https://github.com/radicle-dev/radicle-surf/commit/28ce86151723f79d8d58de57050aabc8d6fe754f